charts/corda/values.schema.json

```
@@ -3045,6 +3060,15 @@
                         },
                         "additionalProperties": false
                     }
+                },
+                "extraArgs": {
+                    "type": "array",
+                    "default": [],
+                    "title": "Allows users to pass any extra arguments to the worker",
```

Everything in this file is about allowing the user to pass something!
```suggestion
                    "title": "Extra command-line arguments for the worker",
```